packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1716,6 +1744,46 @@ module.exports = function(log, config, oauthdb) {
     );
   };
 
+  Mailer.prototype.downloadSubscriptionEmail = async function(message) {
+    const { email, productId, uid } = message;
+
+    log.trace('mailer.downloadSubscription', { email, productId, uid });
+
+    const query = { product_id: productId, uid };
+    const template = 'downloadSubscription';
+    const links = this._generateLinks(
+      this.subscriptionDownloadUrl,
+      email,
+      query,
+      template
+    );
+    const headers = {
+      'X-Link': links.link,
+    };
+    // TODO: product, subject, action and icon must vary per subscription for phase 2
+    const product = gettext('Secure Proxy');
+    const subject = gettext('Welcome to Secure Proxy!');
+    const action = gettext('Download Secure Proxy');
```

Comment:
I figured it was fine to hard-code these for now. When we have more products though, I guess we'll need some kind of lookup operation matching product ids to strings, which sounds like a bit of a nightmare tbh.